### PR TITLE
Execute script lxc-devsetup also with sysvinit and upstart.

### PR DESCRIPTION
**Pull request by clopez**
* This script sets /dev/.lxc which is needed for autodev containers.
  * Previously was only executed with systemd. Execute it also with
    the other init systems (sysvinit and upstart)

### CODE REVIEW

File: config/init/upstart/Makefile.am [hunk 1]
```diff
@@ -1,4 +1,4 @@
-EXTRA_DIST = lxc.conf lxc-instance.conf lxc-net.conf.in
+EXTRA_DIST = lxc.conf.in lxc-instance.conf lxc-net.conf.in
```
Comment (brauner) [inline config/init/upstart/Makefile.am]: This is `lxc.conf` without the `.in`-prefix on purpose; it doesn't need to be processed before.

**Comment by brauner** [comment]
I'll leave this one to @hallyn.

**Comment by hallyn** [comment]
Heh, in general I leave stuff like this to @stgraber .  If he likes I'll look, but this is his domain :)

**Comment by stgraber** [comment]
Oh, I forgot that this feature even still existed, never used it myself but the change seems reasonable.

jenkins: test this please

**Pull request merged**